example/lib/context.dart

```diff
@@ -89,7 +89,7 @@ class WagmiContext {
       return 'wss://polygon-amoy.g.alchemy.com/v2/DynWKvz6PUFaeZNmlxPXNiV1nK4Ac_2D';
```

Comment:
wss is not compatible with transport.http

Reply:
Will resolve it as well in the upcoming commit

Reply:
I will also post a solution here tomorrow where we have implemented using both wss and http together with fallback mechanism as mentioned here - https://wagmi.sh/core/api/transports/fallback